src/service_images/rails/wait-for-it.sh

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
+#   Use this script to test if a given TCP host/port are available
```

Comment:
Same thing here, would be nice to share this file with django(and future frameworks)